### PR TITLE
WD-9628 Refresh core services

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Go to https://ubuntu-com-13767.demos.haus/core/services and compare against [copy doc](https://docs.google.com/document/d/11z973-P-vuaVp7gCQpoQly0gzqNMYX7ETPVghEkDDsE/edit#heading=h.cgeassa57vhx) and design (screenshot below)
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9628

## Screenshots (design review)

core services

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by camillet16** [comment]
@carkod future thanks

Feedback here:

Screenshot 2024-04-22 at 10 55 01 AM
Screenshot 2024-04-22 at 10 55 12 AM
Screenshot 2024-04-22 at 10 55 18 AM
Screenshot 2024-04-22 at 10 55 22 AM
Screenshot 2024-04-22 at 10 55 27 AM
Screenshot 2024-04-22 at 10 55 32 AM
Screenshot 2024-04-22 at 10 56 35 AM

**Comment by juanruitina** [comment]
Looks good, only a minor comment: "We handle the board, you handle the apps" and "Canonical supports Ubuntu Core devices" shouldn't be H3s, just regular paragraphs.

File: templates/core/services/index.html [hunk 1]
```diff
@@ -1,256 +1,294 @@
...
+        <div class="p-section--shallow">
+          <p>
+            Sometimes all you need is a bit of help. We ship open source globally to millions of users every day. Our IoT professional services bring the same infrastructure and expertise to your innovation.
+          </p>
+        </div>
+        <hr />
+        <div class="p-section--shallow">
+          <p>
+            <a class="p-button--positive js-invoke-modal"
+              href="/core/services/contact-us">Contact us</a>
```
Comment (mtruj013) [inline templates/core/services/index.html:33]:
```suggestion
            <a class="p-button--positive"
              href="/core/services/contact-us">Contact us</a>
```

File: templates/core/services/index.html [hunk 2]
```diff
@@ -1,256 +1,294 @@
...
-<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/internet-of-things" data-form-id="1663" data-lp-id="3143" data-return-url="https://ubuntu.com/core/services/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
-</div>
+  <!-- Set default Marketo information for contact form below -->
+  <div class="u-hide"
+       id="contact-form-container"
+       data-form-location="/shared/forms/interactive/internet-of-things"
+       data-form-id="1663"
+       data-lp-id="3143"
+       data-return-url="https://ubuntu.com/core/services/thank-you"
+       data-lp-url="https://pages.ubuntu.com/things-contact-us.html"></div>
```
Comment (mtruj013) [inline templates/core/services/index.html:292]:
```suggestion
```
Looks like it's not supposed to be a modal

**Review by mtruj013 — APPROVED** [review]
LGTM, just one change need

**Comment by camillet16** [comment]
@carkod I missed 2 things on the smaller sizes apologies for the inconvenience.

Here the images need to go above the text
Screenshot 2024-04-24 at 11 56 55 AM
Screenshot 2024-04-24 at 11 56 49 AM

**Pull request merged**